combat-trainer.lic

```diff
@@ -1171,6 +1181,15 @@ class AbilityProcess
         next unless !timer || (Time.now - timer).to_i > 30
         game_state.cooldown_timers[name] = Time.now if activate_khri?(@kneel_khri, name)
       end
+    @barb_buffs.each do |name|
+      if !(Flags["ap-#{name}-active"] == true)
```

Comment:
Is this the same as `if !Flags["ap-#{name}-active"]` or `unless Flags["ap-#{name}-active"]`?